speed_controller:
  ros__parameters:
    odom_topic: /odometry/filtered
    target_topic: /control_target/velocity
    throttle_topic: /autodrive/f1tenth_1/throttle_command
    p: 0.05
    i: 0.01
    d: 0.0
    max_i: 0.25
steer_controller:
  ros__parameters:
    feedback_topic: /autodrive/f1tenth_1/steering
    target_topic: /control_target/steer_angle
    out_topic: /autodrive/f1tenth_1/steering_command
    steer_velocity: 2.0  # radians per second
